### ISSUE TITLE
A silent sign-in request was sent but no user is signed in.

### ISSUE THREAD

**Issue by w595shubham** [AzureAD/microsoft-authentication-library-for-js#2110]
## Library
- [ ] `@azure/msal@1.3.1`
- [ ] `@azure/msal-angular@1.0.0`

## Framework
- [ ] `angular@9.1.9`

## Description
I'm using MSAL library in my project. I'm facing issue in auto token renewal.
When the id token is expired and our application tries to generate access token, it does not refresh the id token and throws, "msal:acquireTokenFailure" event while getting access token.

AADSTS50058: A silent sign-in request was sent but no user is signed in.
Trace ID: 4a70467a-cdac-4edf-88da-04b31ef82800
Correlation ID: 04e87429-0fbe-4d4c-84da-170f290f1738
Timestamp: 2020-08-06 08:33:24Z

We found that, we can subscribe to this failure event using broadcast service and then re-trigger the login process using loginRedirect().

## MSAL Configuration

```js
const protectedResourcesKeysLocal: Map<string, string> = new Map<string, string>();
protectedResourcesKeysLocal.set('<ADMIN_API_ENDPOINT>', '<ADMIN_API_AZURE_CLIENTID>');
protectedResourcesKeysLocal.set('<ENGINE_ENDPOINT>', '<ENGINE_AZURE_CLIENTID>');
protectedResourcesKeysLocal.set('<AUDITSAPI_URL>', '<AZURE_AUDITSAPI_ID>');

const msalConfig: IMsalConfiguration = {
  clientIdKey: '<ADMIN_UI_AZURE_CLIENTID>',
  authorityURLKey: '<AZURE_AUTHORITY_ENDPOINT>',
  azureRedirectURIKey: '<REDIRECT_URI>',
  azureLoggingEnabledKey: '<isLoggingEnabled>',
  extraQueryParametersKey: '<EXTRA_QUERY_PARAMETERS>',
  unprotectedResourcesKeys: [],
  protectedResourcesKeys: protectedResourcesKeysLocal
};
```

## Expected behavior
msal library should handle this case and should redirect user to login screen.
id token and access token should be renewed without throwing any error.

## Browsers/Environment
- [ ] Chrome

**Comment by technical-boy** [comment]
Hi @w595shubham . This is a known issue with msal-angular, which we are planning to address soon. We'll follow up when it has been added to msal-angular.

**Comment by rugglcon** [comment]
@hectormmg any updates on this? Will updating to msal 1.4.0 resolve this issue? We are not using msal-angular, just msal.

**Comment by hectormmg** [comment]
@rugglcon if you are seeing this behavior in msal@1.3.x, I'd expect to see it fixed when we release 1.4.1, based on the changes in #2206.
